--- hibernate-2.1/src/net/sf/hibernate/tool/hbm2ddl/SchemaExport.java	2004-11-23 18:47:40.000000000 +0300
+++ hibernate-2.1-schema-cmt-patch/src/net/sf/hibernate/tool/hbm2ddl/SchemaExport.java	2004-12-17 19:00:48.225084800 +0300
@@ -117,10 +117,6 @@
 				log.info("exporting generated schema to database");
 				connectionProvider = ConnectionProviderFactory.newConnectionProvider(props);
 				connection = connectionProvider.getConnection();
-				if ( !connection.getAutoCommit() ) {
-					connection.commit();
-					connection.setAutoCommit(true);
-				}
 				statement = connection.createStatement();
 			}
 			
